librz/include/rz_core.h

```diff
@@ -784,14 +811,13 @@ RZ_API void rz_core_analysis_hint_list_print(RzAnalysis *a, RzCmdStateOutput *st
 RZ_API int rz_core_analysis_search(RzCore *core, ut64 from, ut64 to, ut64 ref, int mode);
 RZ_API int rz_core_analysis_search_xrefs(RZ_NONNULL RzCore *core, ut64 from, ut64 to);
 RZ_API int rz_core_analysis_data(RzCore *core, ut64 addr, int count, int depth, int wordsize);
-RZ_API void rz_core_analysis_datarefs(RzCore *core, ut64 addr);
-RZ_API void rz_core_analysis_coderefs(RzCore *core, ut64 addr);
-RZ_API RzGraph /*<RzGraphNodeInfo *>*/ *rz_core_analysis_codexrefs(RzCore *core, ut64 addr);
-RZ_API RzGraph /*<RzGraphNodeInfo *>*/ *rz_core_analysis_importxrefs(RzCore *core);
-RZ_API void rz_core_analysis_callgraph(RzCore *core, ut64 addr, int fmt);
+RZ_API RZ_OWN RzGraph /*<RzGraphNodeInfo *>*/ *rz_core_analysis_graph_datarefs(RZ_NONNULL RzCore *core, ut64 addr);
```

Comment:
I'd add also an API that takes a `RzCoreGraphType` as argument and prints the proper graph based on that. This way the cmd handlers can just use that API.

Reply:
It should be an IPI though since there's no use of API function that prints something.

Reply:
Use `rz_core_graph_print`